docs/operate/control/web-app.md

```diff
@@ -36,6 +36,222 @@ Refer to the [Viam TypeScript SDK](https://ts.viam.dev/) documentation for avail
 
 ### Example usage
 
+{{< expand "Example camera and sensor code" >}}
+
+The following files are an example of a simple TypeScript web app that connects to a machine and displays the latestimage from the machine's camera, and the latest sensor readings.
+
+{{<imgproc src="/operate/ts-dashboard.png" resize="x1100" declaredimensions=true alt="A web browser displaying a dashboard with a camera feed and sensor readings." style="max-width:500px" class="imgzoom" >}}
+
+<file>main.ts</file>:
```

Comment:
A common pattern with web dev is to put these three files in tabs. We could do that here to preserve space